model/godleyIcon.cc

```diff
@@ -98,14 +98,14 @@ namespace minsky
     set<string> alreadyAdded;
 
     vars.clear();
-    shared_ptr<GodleyIcon> self;
-    if (auto g=group.lock())
-      self=dynamic_pointer_cast<GodleyIcon>(g->findItem(*this));
+    shared_ptr<GodleyIcon> godleySelf;
+    if (auto g=self->group.lock())
+      godleySelf=dynamic_pointer_cast<GodleyIcon>(g->findItem(*this->self));
```

Comment:
It would be simpler to leave these methods in GodleyIcon, which has a full view of everything.